### DISCUSSION TITLE
Override for App name in navbar

### DISCUSSION THREAD

**Discussion by mmn01-sky** [mealie-recipes/mealie#2343] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
At the top navbar, I would like to be able to change the application name from Mealie to something else e.g. Recipe Manager. 

**(Optional): Suggest A Solution**
An environment variable override for the application name "Mealie" would be really handy nice. 

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature
Unfortunately I don't know how to make this change myself. Sorry.

**Comment by paulthvt** [comment]
I agree, Mealie for non english speakers might not be ideal. But the name is not only visible in the top app navbar. It's also in the app title, in the pwa manifest, etc. It would be nice to be able to customize this name but I guess that it would require some work.

**Comment by Byh0ki** [comment]
I agree too, this would indeed be very nice. My family and friends don't really need/want to know that my receipe website is based on Mealie, it would be nice to be able to have the title (and the thumbnail in site like discord or WhatsApp) to show a customized entry.
